seatunnel-e2e/seatunnel-flink-connector-v2-e2e/pom.xml

```diff
@@ -36,6 +36,7 @@
         <module>connector-iceberg-flink-e2e</module>
         <module>connector-neo4j-flink-e2e</module>
         <module>connector-kafka-flink-e2e</module>
+        <module>connector-elasticsearch-flink-e2e</module>
```

Comment:
Does this code belong to this PR?

Reply:
> > Please add e2e in flink/spark e2e module. So that e2e can be start in CI. <img alt="image" width="1000" src="https://user-images.githubusercontent.com/32387433/200732205-1f4984a8-f704-41e2-bb1e-b22d9a663360.png">
> 
> thx man, done. I have merged upstream dev, and add flink/spark connector-elasticsearch-e2e module.


this code is for this comment. I merge upstream dev, and add elasticsearch-flink-e2e in pom.xml.